src/pages/DelegationCreatePage/DelegationCreate.page.tsx

```diff
@@ -0,0 +1,82 @@
+import { PageContainer, SectionContainer } from '@/components/layout/containers'
+import { useNavigate } from '@/router'
+import { Stack } from '@mui/material'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DelegationCreateCards } from './components/DelegationCreateCards'
+import { StepActions } from '@/components/shared/StepActions'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import { match } from 'ts-pattern'
+import { DelegationCreateForm } from './components/DelegationCreateForm'
+
+/***
+ * It shows the cards component to choose delegation kind or the form component based on the state KIND or FORM
+ */
+export const DelegationCreatePage: React.FC = () => {
+  const { t } = useTranslation('party')
+  const navigate = useNavigate()
+  const [delegationKind, setDelegationKind] = useState<'CONSUME' | 'PROVIDE' | ''>('')
+  const [activeStep, setActiveStep] = useState<'KIND' | 'FORM'>('KIND')
+
+  return (
+    <PageContainer
+      title={t('delegations.create.title')}
+      backToAction={{
+        label: t('delegations.actions.backToTenant'),
+        to: 'TENANT',
+      }}
```

Comment:
I think this should bring the user back to the delegations list? 
I know the design is like this, but it does not make sense, I would change it